site.yml

```diff
@@ -364,6 +364,7 @@
     - role: 'openstack-network'
       tags: ['openstack', 'setup', 'openstack-network']
   environment: "{{ env_vars|default({}) }}"
+  when: neutron.enable_external_interface|bool
```

Comment:
Move this when to the role statement, rather than the play. Conditionals can apply to roles, but not plays.